### ISSUE TITLE
The onchange doesnt work?

### ISSUE THREAD

**Issue by rohitkrishna094** [vayser/react-js-pagination#88]
When I click the last page or first page or previous page or next page `li`'s, the onChange is passing a pageNumber which is wrong. Is there a handler for first page, last page, previous page and next page?

**Comment by hassaan-prog** [comment]
i am facing the same issue, when i redirect user from create or update to list (in the middle pages of pagination). it does not go from middle pages to first page but allows us to navigate to the last.

**Comment by rohitkrishna094** [comment]
Yeah, I think the handler should return something different instead of pageNumber. I ended up writing my own pagination component, but I think this is a small fix in this library if author wants. Or maybe there is a way to handle it in this library which we are not aware of.
